ca_tools.erl

```diff
@@ -1,40 +1,42 @@
 -module(ca_tools).
--export([iterate/3, make_rule/1, make_state/1, random_state/1]).
+-export([start/4]).
 
--spec iterate(ca:state(), [ca:rule()], non_neg_integer()) -> ca:state().
-iterate(State, _, 0) ->
+-type automaton() :: elementary | von_neumann.
+-type state_type() :: random | origin.
+
+-spec start(automaton(), {state_type(), integer()}, integer(), non_neg_integer()) -> ca:state().
```

Comment:
Line 7 is too long: -spec start(automaton(), {state_type(), integer()}, integer(), non_neg_integer()) -> ca:state()..